### PR TITLE
feat: [CDS-40604]: pdc connectors UI improvements

### PR DESCRIPTION
**Pull request by radujomir**
### Summary

PDC connector adjustments

#### Screenshots

Before:
Screenshot 2022-08-04 at 11 48 03

After:
Screenshot 2022-08-04 at 11 47 08

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by radujomir** [comment]
retrigger titlecheck

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 31 Issues 
* Code issue 31 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10298)

File: src/modules/35-connectors/components/CreateConnector/PdcConnector/StepDetails/PdcDetails.tsx [hunk 1]
```diff
@@ -84,15 +118,16 @@ const PdcDetails: React.FC<StepProps<StepConfigureProps> & Partial<PdcDetailsPro
...
                 <UploadJSON
                   setJsonValue={json => {
                     setHostsJSON(json)
-                    formikProps.setFieldValue('hosts', json.map(hostItem => hostItem.hostname).join('\n'))
                   }}
                 />
-              </Layout.Horizontal>
+              )}
             </Container>
+            <Container></Container>
```
Comment (upadhyay-an) [inline src/modules/35-connectors/components/CreateConnector/PdcConnector/StepDetails/PdcDetails.tsx:130]: A blank container, is it intentional?

Comment (radujomir) [reply src/modules/35-connectors/components/CreateConnector/PdcConnector/StepDetails/PdcDetails.tsx:130]: yes this was intentional to align items

File: src/modules/35-connectors/pages/connectors/views/savedDetailsView/SavedConnectorDetails.tsx [hunk 2]
```diff
@@ -968,6 +968,18 @@ export const RenderDetailsSection: React.FC<RenderDetailsSectionProps> = props =
   )
 }
 
+const getPDCConnectorHosts = (connector: ConnectorInfoDTO) => {
+  const hosts = connector.spec?.hosts.map((host: any) => host.hostname)
+  return hosts && hosts.length
```
Comment (upadhyay-an) [inline src/modules/35-connectors/pages/connectors/views/savedDetailsView/SavedConnectorDetails.tsx:973]: Nit: ```Array.isArray(hosts)```

**Review by upadhyay-an — APPROVED** [review]

**Review by litvinovstas — APPROVED** [review]

**Review by litvinovstas — APPROVED** [review]

**Review by upadhyay-an — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**